src/main/java/Main.java

```diff
@@ -0,0 +1,12 @@
+public class Main {
+    public static void main(String[] args) {
+        Lottery firstRandomBall = new Lottery();
+        Lottery secondRandomBall = new Lottery();
+        Lottery thirdRandomBall = new Lottery();
+
+        System.out.println(firstRandomBall.getRandomBall());
+        System.out.println(secondRandomBall.getRandomBall());
+        System.out.println(thirdRandomBall.getRandomBall());
+
```

Comment:
we need to create only one lottery instance, and then we call its method in a for loop